documents/DeveloperGuide/CodeExamples.md

```diff
@@ -93,8 +93,8 @@ Module().then((_module) => {
 
     // Create a simple shader interface.
     const simpleSrf = doc.addNodeDef('ND_simpleSrf', 'surfaceshader', 'simpleSrf');
-    const diffColor = simpleSrf.setInputValuecolor3('diffColor', new mx.Color3(1.0, 1.0, 1.0));
-    const specColor = simpleSrf.setInputValuecolor3('specColor', new mx.Color3(0.0, 0.0, 0.0));
+    const diffColor = simpleSrf.setInputValueColor3('diffColor', new mx.Color3(1.0, 1.0, 1.0));
```

Comment:
Please undo the changes in this file. This is the documentation for C++ :)
Btw, this made me notice that we're now diverging from the C++ interface, by renaming these functions...

Reply:
Ah, I'm sorry :D